src/grid/styles/grid.scss

```diff
@@ -50,6 +50,27 @@
         }
     }
 
+    .thy-grid-click-display-operation {
```

Comment:
thy-grid-click-display-operation change to 'thy-grid-item-hover-display-operation'